http/src/client/builder.rs

```diff
@@ -82,13 +84,13 @@ impl ClientBuilder {
         self
     }
 
-    /// Set the reqwest client to use.
+    /// Set a pre-configured reqwest client builder to build off of
     ///
-    /// All of the settings in the client will be overwritten by the settings
-    /// in this configuration, if specified.
+    /// The proxy and timeout settings in the reqwest client will be overridden by
+    /// those in this builder
     ///
     /// The default client is a RusTLS-backed client.
```

Comment:
I know this was here before, but can this be made better?
```suggestion
    /// The default client uses Rustls as its TLS backend.
```